vars/dockerExecute.groovy

```diff
@@ -1,51 +1,78 @@
 import com.cloudbees.groovy.cps.NonCPS
+import com.sap.piper.ConfigurationLoader
+import com.sap.piper.ConfigurationMerger
 
 def call(Map parameters = [:], body) {
 
     def STEP_NAME = 'dockerExecute'
     def PLUGIN_ID_DOCKER_WORKFLOW = 'docker-workflow'
 
-    handlePipelineStepErrors(stepName: STEP_NAME, stepParameters: parameters){
-        def dockerImage = parameters.dockerImage ?: ''
-        Map dockerEnvVars = parameters.dockerEnvVars ?: [:]
-        def dockerOptions = parameters.dockerOptions ?: ''
-        Map dockerVolumeBind = parameters.dockerVolumeBind ?: [:]
+    handlePipelineStepErrors(stepName: STEP_NAME, stepParameters: parameters) {
+        final script = parameters.script
+        Map stepConfig = ConfigurationLoader.stepConfiguration(script, 'kubernetes')
 
-        if(dockerImage) {
+        Set parameterKeys = ['dockerImage',
```

Comment:
Use fields to have the keys more readable at the top of the step, see https://github.com/SAP/jenkins-library/blob/master/vars/snykExecute.groovy#L10